### DISCUSSION TITLE
Add support for Azure Repos as a source control provider

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cloud#32] [Feature Requests & Enhancements]
https://azure.microsoft.com/en-us/products/devops/repos/

Would only need to support the source control part - i.e. nothing else under the Azure DevOps umbrella.

**Comment by angrybrad** [comment]
Can only find one other app that appears to have done this:
https://docs.fossa.com/docs/azure-repos

Docs: https://learn.microsoft.com/en-us/azure/devops/repos/?view=azure-devops

**Reply by timkelty** [reply]
AND THEN THERE WERE 2

**Reply by angrybrad** [reply]
#trendsetters

**Comment by angrybrad** [comment]
Craft Cloud now supports Azure Repos!
